addons/im_livechat/views/im_livechat_channel_templates.xml

```diff
@@ -10,6 +10,7 @@
                 <script type="text/javascript" src="/im_livechat/static/src/js/im_livechat_channel_form_view.js"></script>
                 <script type="text/javascript" src="/im_livechat/static/src/js/im_livechat_channel_form_controller.js"></script>
                 <script type="text/javascript" src="/im_livechat/static/src/bugfix/bugfix.js"></script>
+                <script type="text/javascript" src="/im_livechat/static/src/components/composer/composer.js"></script>
```

Comment:
We can't add new file in stable, but this should actually target master. This issue is there since forever, no need to try to fix in 14.0.